simpeg_drivers/components/factories/simpeg_factory.py

```diff
@@ -57,6 +57,7 @@ class SimPEGFactory(ABC):
         "induced polarization 2d",
         "induced polarization pseudo 3d",
         "fdem",
+        "fem",
```

Comment:
why support this?

Reply:
If it's to support older ui.json files, can we use a validator on the options class instead?

Reply:
..oh, you already did that!  You should be able to remove the "fem" entry here then, right?

Reply:
yep